bin/dbsbuffer-file-fix.py

```diff
@@ -9,7 +9,8 @@
 from WMCore.WMInit import connectToDB
 
 def fixDBSmissingFileAssoc():
-    os.environ['WMAGENT_CONFIG'] = '/data/srv/wmagent/current/config/wmagent/config.py'
+    os.environ.get('WMA_CONFIG_FILE', '/data/srv/wmagent/current/config/config.py')
```

Comment:
Here we are not getting the variable, but setting it, so please use `os.environ['WMAGENT_CONFIG'] = ...`